frontend/src/metabase/home/components/HomeContent/HomeContent.unit.spec.tsx

```diff
@@ -181,69 +181,4 @@ describe("HomeContent", () => {
       screen.queryByText(/Here are some explorations/),
     ).not.toBeInTheDocument();
   });
-
-  describe("embed-focused homepage", () => {
```

Comment:
Those tests are about the minimal version, which is removed in this pr.
The tests checking the full version are in the next pr.